src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +58,29 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        final MovieSessionService movieSessionService =
+                (MovieSessionService) INJECTOR.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        final AuthenticationService authenticationService =
+                (AuthenticationService) INJECTOR.getInstance(
+                        AuthenticationService.class);
+        final User bob = new User(BOB_EMAIL, BOB_PASSWORD);
+        final User alice = new User(ALICE_EMAIL, ALICE_PASSWORD);
+        final User unknown = new User(BOB_EMAIL, ANOTHER_PASSWORD);
+
+        System.out.println(
+                authenticationService.register(bob.getEmail(), bob.getPassword()));
```

Comment:
Ensure that the password is hashed before storing the user in the `register` method. This is crucial for security purposes.